CONTRIBUTING.md

```diff
@@ -72,4 +76,4 @@ The following rule governs documentation contributions:
 
 * When creating an issue, try using one of our issue templates which already contain some guidelines on which content is expected to process the issue most efficiently. If no template applies, you can of course also create an issue from scratch.
 
-* Please apply one or more applicable [labels](https://github.com/corona-warn-app/cwa-app-android/labels) to your issue so that all community members are able to cluster the issues better.
\ No newline at end of file
+* Please apply one or more applicable [labels](https://github.com/corona-warn-app/cwa-app-android/labels) to your issue so that all community members are able to cluster the issues better.
```

Comment:
Are you aware that only project members can add and remove labels on github?
Asking general github users/"the community" to apply labels is therefor a bit pointless.

Reply:
Thanks for spotting that. We'll check this sentence in the other repos, too.